src/app/components/splash-screen/splash-screen.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnInit, OnDestroy , ViewEncapsulation } from '@angular/core';
+import { UtilityService } from 'src/app/services/utility.service';
+import {DEFAULT_IMAGE_TEXT} from "src/constants/constant";
+
+@Component({
+  selector: 'app-splash-screen',
+  templateUrl: './splash-screen.component.html',
+  styleUrls: ['./splash-screen.component.scss'],
+  encapsulation: ViewEncapsulation.None
+})
+export class SplashScreenComponent implements OnInit , OnDestroy {
+  showSplash: boolean;
+  imgText= DEFAULT_IMAGE_TEXT;
+  timeOutID: number;
```

Comment:
field could be [private](https://www.typescriptlang.org/docs/handbook/2/classes.html#private).